Feature/update checkout screen

HenryTNH opened the pull request.

HenryTNH committed:
Add .idea file to .gitignore

HenryTNH committed:
Update floating action button icon

HenryTNH committed:
Add PurchaseFragment4

HenryTNH committed:
Add PurchaseFragment4 to ShoppingCartActivity

HenryTNH committed:
Modify logic of PurchaseFragment3

HenryTNH committed:
Modify purchase 1 layout

HenryTNH committed:
Modify purchase 2 layout

HenryTNH committed:
Modify purchase 3 layout

HenryTNH committed:
Modify purchase 4 layout

StevenKo commented:
![device-2016-04-18-165701](https://cloud.githubusercontent.com/assets/1100488/14598948/5375a2c0-0587-11e6-8233-c7c5b4dbd001.png)
![device-2016-04-18-165835](https://cloud.githubusercontent.com/assets/1100488/14598949/539a48d2-0587-11e6-8123-76828ad336e7.png)

選擇取貨的方式，全家通昌店，下一步的按鈕太小了，再做大一點，跟"登入FB購買"一樣大

HenryTNH committed:
Modify button size

HenryTNH committed:
Modify thank you message

HenryTNH committed:
Add tab layout

StevenKo commented:
![device-2016-04-20-091559](https://cloud.githubusercontent.com/assets/1100488/14660551/e8ace630-06d8-11e6-8264-ef054f6ffd87.png)
圖片有時會變正方形

HenryTNH committed:
Fix image ratio problem

StevenKo commented:
在 create user 時，也把 email 傳上來，會再加 email 欄位
![screen shot 2016-04-21 at 6 36 25 pm](https://cloud.githubusercontent.com/assets/1100488/14706274/3bf45978-07f0-11e6-8c98-8a860440a457.png)

HenryTNH committed:
Update facebook-android-sdk

HenryTNH committed:
Add test host

HenryTNH committed:
Add email to user data and post to server

HenryTNH committed:
Add ship_email to order and post order with guest email

HenryTNH committed:
Fix purchase layout problem

1. Fix FB login from purchaseFragment1 to purchaseFragment2, email layout still visible problem.

1. Fix FB login from purchaseFragment1 to purchaseFragment2 then press back, "not login layout" still visible problem.
